ppr-ui/src/components/parties/Parties.vue

```diff
@@ -70,9 +71,10 @@
       <v-col v-else>
         <v-card flat class="add-party-container mt-2 mb-8">
           <div class="px-6 pt-8">
-            <h3 class="pb-2">Change Registering Party</h3>
+            <h3 v-if="!isSbc" class="pb-2">Change Registering Party</h3>
             <span class="body-text">
-            Change the Registering Party by entering the registering party code
+            <span v-if="!isSbc">Change</span><span v-else>Include</span> the Registering Party
```

Comment:
instead of the spans you can also do: `{{ !isSbc ? 'Change' : 'Include' }}`

Reply:
done